migrations/20231102101356_user.py

```diff
@@ -23,14 +57,64 @@ def upgrade(db: "pymongo.database.Database"):
                 "_id": user['_id'],
                 "email": user['profile']['email'],
                 "hashed_password": user['profile']['hashed_password'],
-                "is_active": True,
-                "is_superuser": False,
-                "is_verified": False,
+                "is_active": user['active'],
+                "is_superuser": is_superuser,
+                "is_verified": is_superuser,
```

Comment:
I think this should only be enabled for the initial admin user. Not all the admin users should be automatically verified.

Reply:
In the former user schema, we didn't have any mechanism to verify the user. So, basically, with the migrations, we should make all the users unverified. As a part of adopting the process of a new schema, the user should undertake verification even if it's the initial admin user.  So, we should use `"is_verified":  False` in the migration.
What do you think?